src/sbt-test/interplay/library-with-plugin/test

```diff
@@ -7,26 +7,30 @@ $ exec git add .
 $ exec git commit -m commit
 
 # Setup remote git repo in target directory
-$ exec git init --bare target/remote
-$ exec git remote add origin target/remote
+$ exec git init --bare $PWD/target/remote
```

Comment:
Yep.

What I'm doing (not pushed yet) is something like:

```
 > someContains mock-sbt-plugin/target/scala-2.10/sbt-0.13/publish-version mock-sbt-plugin/target/scala-2.12/sbt-1.0/publish-version Bintray-Sbt-Publish-playframework-sbt-plugin-releases-mock:1.2.3
```
In other words, I'm trying to check both combinations here (sbt 0.13 with Scala 2.10 and sbt 1.0 with Scala 2.12). If none is found, then the test fails. I couldn't find a way to do something like:

```
contains mock-sbt-plugin/target/scala-${scala.version}/sbt-${sbt.version}/publish-version Bintray-Sbt-Publish-playframework-sbt-plugin-releases-mock:1.2.3
```
Of course, I can do it myself, but I was wondering if scripted plugin has something out-of-the-box.